File: soal4.pas
#include <stdio.h>

int main() {
    int a, b, c, temp;

    printf("Masukkan bilangan pertama: ");
    scanf("%d", &a);
    printf("Masukkan bilangan kedua: ");
    scanf("%d", &b);
    printf("Masukkan bilangan ketiga: ");
    scanf("%d", &c);

    if (a > b) {
        temp = a; a = b; b = temp;
    }
    if (a > c) {
        temp = a; a = c; c = temp;
    }
    if (b > c) {
        temp = b; b = c; c = temp;
    }

    printf("Urutan: %d %d %d\n", a, b, c);

    return 0;
}

File: soal4.c
#include <stdio.h>

int main() {
    int a, b, c;
    int kiri, kanan;

    printf("Masukkan sisi a: ");
    scanf("%d", &a);
    printf("Masukkan sisi b: ");
    scanf("%d", &b);
    printf("Masukkan sisi c: ");
    scanf("%d", &c);

    kiri = a*a + b*b;
    kanan = c*c;

    if (kiri == kanan)
        printf("Segitiga siku-siku\n");
    else if (kiri > kanan)
        printf("Segitiga lancip\n");
    else
        printf("Segitiga tumpul\n");

    return 0;
}